.github/workflows/build-and-deploy.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         id: docker-build-push
         with:
           team: teamfamilie
+          tag: latest
+          push_image: true
+          dockerfile: Dockerfile
+          docker_context: .
+          byosbom: build/reports/bom.json
```

Comment:
Hva gjør dette? Hvorfor ønsker vi dette framfor deafult?

Reply:
Dette er default verdier, lettere å se ha som kan skrus på, men trengs ikke.

Reply:
Eller denne var ikke default nei 👍 den kan vi fjerne

Reply:
Hvordan skjønner man det? Ikke så lett å lese fra dokumentasjonen. 
> \# defaults to use Trivy for SBOM generation, if salsa is true, but can be overwritten sending in a path to a  pre-generated SBOM

Hva om nais ønsker å endre på hvordan de genererer SBOM i framtiden? Tenker det er mindre å maintaine om vi bare går for default